src/field_theory/intermediate_field.lean

```diff
@@ -331,6 +331,13 @@ begin
     rw [← aeval_def, aeval_coe, aeval_def, hProot, coe_zero] },
 end
 
+/-- Let `L/K` be a field extension, and let `E ≤ F` be intermediate fields, where `hEF` is a term
+  of type `E ≤ F`. Then `intermediate_field.inclusion hEF` is the `K`-algebra homomorphism
+  `E →ₐ[K] F` given by inclusion. -/
```

Comment:
This is quite wordy. There's no need for precise language introducing every variable, as lean already does that for us. Using the `subalgebra` one as a template:
```suggestion
/-- The map `E → F` when `E` is an intermediate field contained in the intermediate field `F`.

This is the intermediate field version of `subalgebra.inclusion`. -/
```